### PR TITLE
Implement your portfolio panel UI

### PR DESCRIPTION
**Pull request by ivanng-oursky**
~~BLOCKED BY oursky/likedao#131~~

## Changes
- Created and added your portfolio panel in portfolio screen
- Integrate your portfolio API with UI

### Desktop

image

### Mobile

image

### Wallet Not Connected

image

refs oursky/likedao#141

### CODE REVIEW

File: react-app/src/components/PortfolioScreen/PortfolioScreen.tsx [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
 
-  useEffect(() => {
-    console.log(requestState);
-  }, [requestState]);
+  useEffectOnce(
+    () => {
+      if (isRequestStateError(requestState)) {
+        toast.error("Failed to fetch Portfolio");
+      } else if (isRequestStateLoaded(requestState) && !requestState.data) {
+        toast.error("No data fetched for portfolio");
```
Comment (hochiw) [inline react-app/src/components/PortfolioScreen/PortfolioScreen.tsx:21]: i18n?

File: react-app/src/components/PortfolioScreen/PortfolioPanel.tsx [hunk 2]
```diff
@@ -9,7 +9,53 @@ interface PortfolioPanelProps {
...
+    <div className={cn("flex", "justify-center", "mb-9")}>
+      {profilePicture ? (
+        <img
+          className={cn(
+            "rounded-full",
+            "w-[120px]",
+            "h-[120px]",
+            "sm:w-[180px]",
+            "sm:h-[180px]",
+            "sm:mr-9",
```
Comment (hochiw) [inline react-app/src/components/PortfolioScreen/PortfolioPanel.tsx:36]: I think if a margin is against a component that is not part of this component then i think it should be handled at the same level where both of these components are placed in, i.e should pass a className to the outer container, which applies for the 'mb-9' as well.

File: react-app/src/components/PortfolioScreen/PortfolioPanel.tsx [hunk 3]
```diff
@@ -78,6 +88,24 @@ const PortfolioPanel: React.FC<PortfolioPanelProps> = ({ portfolio }) => {
       </div>
       <div className={cn("mt-11", "mb-6", "sm:flex")}>
         <ProfilePicture profile={portfolio.profile} />
+
+        <div className={cn("flex", "flex-col", "items-start")}>
+          <p className={cn("text-xl", "leading-6", "font-medium", "mb-3")}>
+            {portfolio.profile?.dtag ?? truncateAddress(portfolio.address)}
```
Comment (hochiw) [inline react-app/src/components/PortfolioScreen/PortfolioPanel.tsx:123]: doesn't seem like this component uses middle ellipsis

**Review by hochiw — CHANGES_REQUESTED** [review]

File: react-app/src/components/PortfolioScreen/PortfolioPanel.tsx (hunk 3 above)
Comment (hochiw) [reply react-app/src/components/PortfolioScreen/PortfolioPanel.tsx:123]: nvm i see wrong

**Comment by ivanng-oursky** [comment]
@hochiw updated 🙇‍♂️

**Review by hochiw — APPROVED** [review]

**Pull request merged**